scripts/installation/installROOT.sh

```diff
@@ -34,12 +34,11 @@ tar -xvzf $f
 rm -rf source
 mv root-$ROOT_VERSION source
 
-mkdir -p ${ROOT_DIR}/build
-cd ${ROOT_DIR}/build
+mkdir -p ${ROOT_DIR}/root_build
+cd ${ROOT_DIR}/root_build
```

Comment:
Why we need to change to `root_build`? Inside the `root-6.26.10` directory there will be 3 directories, `build` `install` and `source`.